src/rootfs/files/configs/etc/super/var/lib/rancher/rke2/server/manifests/k8s.yaml

```diff
@@ -985,7 +985,7 @@ spec:
                 echo "$filtered_configmap" | kubectl apply -n $NAMESPACE -f -
               fi
             else
-                # created by tools/osbuilder/rootfs-builder/ubuntu/superprotocol/check-config-files.sh
+                # created by tools/osbuilder/rootfs-builder/ubuntu/superprotocol/check-config-files.sh (deleted)
```

Comment:
I guess this logic is redundant, as `check-config-files.sh` currently doesn't exist. It's unnecessary, as this logic appears to be incorrect for multi-node K8s configurations.